Userland/Libraries/LibVideo/Containers/Matroska/Reader.cpp

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2021, Hunter Salyer <thefalsehonesty@gmail.com>
- * Copyright (c) 2022, Gregory Bertilson <Zaggy1024@gmail.com>
+ * Copyright (c) 2023, Gregory Bertilson <Zaggy1024@gmail.com>
```

Comment:
We add to the year, rather than replace it :)
```c++
2022-2023
```

Reply:
Ah, whoops, that was just a brain fart :^)